### ISSUE TITLE
Please make sure BB Themer's header has the same options available as the BB Theme's header

### ISSUE THREAD

**Issue by Lenitas** [beaverbuilder/feature-requests#170]
I use the BB theme's header customizations most of the time, but once in a while I rely on Themer to build a better, more customized header, like adding a button or shape border or partial transparency, or using the excellent conditional logic features.

However, when using Themer's header layouts, some key options are missing, such as "Hide Header Until Scroll".

With some custom CSS/javascript gymnastics you can make it work, however not to the point where you can enter the number of pixels until the header shows up.

It would be great to be able to switch to a Themer header without losing access to options that the BB theme provides.

**Comment by studioavanti** [comment]
Agreed, and especially with the Themer Header "Overlay + Sticky + Shrink" options enabled, make possible to: 
• Change logo on scroll
• Change text color on scroll

Use case : a header overlaying a dark background on page load needs a white logo and wihite text, and a colored logo (different logo file) and black text when the header background turns white on scroll.

**Comment by A-Buzon** [comment]
This really, really, really needs to happen.  Pretty please!?  

Though #243 is first on my wishlist though!
